File: src/main/java/project/rpg/effect/Burns.java
package project.rpg.effect;

import org.bukkit.entity.LivingEntity;

public class Burns extends EffectBase {

    @Override
    public void effect() {
        this._entity.setFireTicks(this._second*20);
    }

    public Burns(LivingEntity entity, int second) {
        super(entity, second);
    }

}

File: src/main/kotlin/project/rpg/effects/objects/Burns.kt
package project.rpg.effects.objects

import org.bukkit.entity.LivingEntity
import project.rpg.effects.EffectBase

class Burns(entity: LivingEntity, second: Int) : EffectBase(entity, second) {
    override fun effect(durationMillis: Long) {
        entity.fireTicks = second * 20
    }
}
